evm/utils/keccak.py

```diff
@@ -10,4 +10,9 @@ def keccak(value):
     return keccak_256(value).digest()
 
 
+# FIXME: this is a terrible name.
+def keccak_updateable(value):
```

Comment:
Thinking that it might be better to just use `keccak_256` directly from the `sha3` library instead of this proxy.

Reply:
Agreed, I'll do that